src/Components/News.js

```diff
@@ -76,19 +76,31 @@ export class News extends React.Component {
       loading: false,
     };
   }
+
+  async componentDidMount(){
+    console.log("cdm1");
+    let url =" https://newsapi.org/v2/everything?q=tesla&from=2022-09-06&sortBy=publishedAt&apiKey=ebf7822aea2f4a3baec55f66d15e5926"
+    let data = await fetch(url);
+    let parseData = await data.json()
```

Comment:
Why do you need await at this line

Reply:
since .json is an async function that's why we need await  here '
For reference I used the below link .
https://stackoverflow.com/questions/59555534/why-is-json-asynchronous#:~:text=json()%20operation%20ends%20up,incoming%20stream%20that%20is%20asynchronous